eng/stages/custom-publish.yml

```diff
@@ -2,128 +2,44 @@ parameters:
   dependsOnPublishStages: []
 
 stages:
-# Create one stage per pipeline that needs some extra publish steps.
+
+# Stages-based publishing entry point
+- template: /eng/common/templates/post-build/post-build.yml
```

Comment:
The ordering here is pretty confusing to me.  I think it would be better to declare the staging in the dependency order.

```
foreach
  CustomPublishBlob
post-build
foreach
  CustomPublishFinal